geoips/plugins/modules/colormaps/tpw/tpw_purple.py

```diff
@@ -48,7 +48,11 @@ def call():
     mpl_colors_info = {
         "cmap": from_ascii(
             pathjoin(
-                gpaths["BASE_PATH"], "image_utils", "ascii_palettes", "tpw_purple.txt"
+                gpaths["BASE_PATH"],
+                "plugins",
+                "txt",
+                "ascii_palettes",
+                "tpw_purple.txt",
```

Comment:
This should probably use `geoips_utils.find_ascii_palette()`?

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/226
@jsolbrig Please resolve if you are good with this.

Reply:
I think this still should be find_ascii_palette, shouldn't it? As it is, it won't find ascii files from plugin packages.

Reply:
As @mindyls pointed out to me in slack, this is a palette that is explicitly in GeoIPS, so this does work.